openapi/openapi.yaml

```diff
@@ -629,6 +629,22 @@ components:
           description: The type (or types) of descriptors available.
           items:
             $ref: "#/components/schemas/DescriptorType"
+        descriptor_type_version:
+          type: object
+          description: A map providing information about the language versions used in this tool. The keys should be the
+            same values used in the `descriptor_type` field, and the value should be an array of all the language versions used 
+            for the given `descriptor_type`. Depending on the `descriptor_type` (ex CWL) multiple version values may be used
+            in a single tool.
+          example: |
+            {
+              "WDL": ["1.0"],
+              "CWL": ["v1.0.2"],
```

Comment:
Even though it's obvious because it's an array, an example with two versions would be nice.